### DISCUSSION TITLE
docker 环境下编译0.15.0失败

### DISCUSSION THREAD

**Discussion by ljmat** [apache/doris#7653] [A - General / Q&A] [answered]
/root/apache-doris-0.15.0-incubating-src/be/src/util/aes_util.cpp:105:20: error: aggregate 'EVP_CIPHER_CTX aes_ctx' has incomplete type and cannot be defined

**Comment by BiteTheDDDDt** [comment]
请尝试更新三方库到最新版本

**Comment by ljmat** [comment, marked as answer]
编译0.15.0需要使用apache/incubator-doris:build-evn-for-0.15.0版本的镜像。
